docs/source/test_raw.md

```diff
@@ -0,0 +1,25 @@
+# Test
+
+
+```python
+from manim import *
+
+class TestScene(Scene):
+    def construct(self):
+        dot = Dot()
+        self.add(dot)
+        self.wait(1)
+```
+
+
+and one more:
+
+```python
+from manim import *
+
+class TestSceneXX(Scene):
+    def construct(self):
+        sq = Square()
+        self.add(sq)
+        self.wait(1)
+```
```

Comment:
Will this be a part of the final documentation? It's just for our internal testing right?

Is there any way to prevent sphinx from taking this file into account when making "production" documentation?

Reply:
This is only for testing.
And to the second question: 
This might be possible, but I don't know how. 
Maybe we can stay with one markdown file, where the gifs become inserted.

Reply:
If this is only for (local?) testing, let's not add it to the repository at all.

Reply:
Yes, I will remove it soon.